internal/aks/delete.go

```diff
@@ -0,0 +1,69 @@
+package aks
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2020-11-01/containerservice"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-10-01/resources"
+	"github.com/Azure/go-autorest/autorest/to"
+	aksv1 "github.com/rancher/aks-operator/pkg/apis/aks.cattle.io/v1"
+	"github.com/sirupsen/logrus"
+)
+
+func RemoveResourceGroup(ctx context.Context, groupsClient *resources.GroupsClient, spec *aksv1.AKSClusterConfigSpec) error {
```

Comment:
This isn't used anywhere now

Reply:
Ok, I wanted to keep it, for future use cases, but yeah let's clean it up